Automattic/jetpack
# Remove duplicate views: Enforce WP Admin for Pages

Fixes https://github.com/Automattic/wp-calypso/issues/95620

## Proposed changes:

Enforces the WP Admin of Pages for users participating in the "Remove duplicate views" experiment.

<img width="1280" alt="Screenshot 2025-01-09 at 14 42 42" src="https://github.com/user-attachments/assets/07bfc867-7d42-4695-b3d1-c29ccdae0711" />


### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

- Visit the experiment page: 22129-explat-experiment
- Assign yourself to the treatment variant
- Go to wordpress.com and select a site with the default admin interface
- Make sure the "Pages" menu links to WP Admin
- Make sure you see an educational notice the first time you visit the WP Admin view
